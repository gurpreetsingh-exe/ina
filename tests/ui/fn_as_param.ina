// COMPILE_FAIL

fn f(f0: fn()) {
    f0();
}

fn g(_: i32) {}

fn main() {
    f(g);
}

