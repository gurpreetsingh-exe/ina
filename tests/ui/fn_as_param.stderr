error: mismatch types
   --> fn_as_param.ray:10:7
    |
 LL |     f(g);
    |       ^ expected `fn() -> ()`, found `fn(i32) -> ()`

